workflow-v2.0.0/README.md

```diff
@@ -0,0 +1,16 @@
+# Workflow v2.0.0
+
+Please report any issues you find in testing Workflow to the appropriate GitHub repository:
```

Comment:
Should we change `testing Workflow` to `running Workflow` -- or a different verb other than `testing`?